jurry/amqp-symfony-bundle/1.0/config/packages/jurry_rabbitmq.yaml

```diff
@@ -0,0 +1,15 @@
+amqp_handler:
+#    You can define connection in both ways: as array or as a string
+#    connection: 'tcp://guest:guest@172.18.0.01:5672'
```

Comment:
Maybe use a string dsn instead and pass it in via `.env`?

Reply:
Developer can define the connection in both ways, as a string or as an array. This increases the flexibility.

Reply:
If both are equivalent how does it make it more flexible? Currently "best practice" is to use a single DSN that is set via .env (see Doctrine, Messenger, Mailer), instead of having 4 different parameters (host, port, user, password) that are set independently thus increasing the chance of errors. If a bundle supports a dsn I think for a consistent DX it makes sense to use that instead.

Reply:
Ok, will do

Reply:
Connection now is handled always as string